playbooks/roles/backups/files/backup.py

```diff
@@ -316,8 +344,10 @@ def _main():
     restore_path = args.restore_path
     s3_id = args.s3_id or os.environ.get('BACKUP_AWS_ACCESS_KEY_ID')
     s3_key = args.s3_key or os.environ.get('BACKUP_AWS_SECRET_ACCESS_KEY')
+    azure_account = args.azure_account or os.environ.get('BACKUP_AZURE_STORAGE_ACCOUNT')
+    azure_key = args.azure_key or os.environ.get('BACKUP_AZURE_STORAGE_KEY')
```

Comment:
I've considered using env vars as fallbacks in ax as well bud I've decided that it's not really useful all that much. One can automate either way (ie. not requiring user input) but flags make it explicit.